angularjs-portal-frame/src/main/webapp/index.html

```diff
@@ -18,6 +18,7 @@
 </head>
 
 <body ng-controller="MainController as mainCtrl">
+<noscript>The new MyUW requires JavaScript to run. Please enable it to fully utilize this website. Please contact the help desk with any questions/concerns.</noscript>
```

Comment:
Do we believe the old MyUW doesn't require `JavaScript`?  If so worth giving the view an invitation to "Try to use Classic MyUW, which may work better than new MyUW without JavaScript."?  If not, worth making this just "MyUW requires JavaScript..." so as to not raise false hopes?